config/routes.rb

```diff
@@ -11,4 +11,10 @@
   # Defines the root path route ("/")
   root to: "technicians#index"
   resources :technicians, only: [:index, :new, :create,:show, :update,:destroy]
+  resources :users do
+    resources :appointments
+  end
+  resources :technicians do
+    resources :appointments
+  end
```

Comment:
Check well this route and see how  you can refactor it with one on line 13 because is creating duplicate ruotes

Reply:
Don't you think the duplication is necessary? Since we want to get appointments through technicians and through ordinary users too.

Reply:
Don't think so because once you execute r`ails routes` you are getting the technicians twice

Reply:
@Bateyjosue that is because the technicians routes are defined twice.

Reply:
What he needs to do is remove the code on line 13